lua/lspsaga/symbolwinbar.lua

```diff
@@ -28,11 +28,14 @@ local function respect_lsp_root(buf)
   if #clients == 0 then
     return
   end
-  local root_dir = clients[1].config.root_dir
+  local root_dir = clients[1].config.root_dir or vim.fn.getcwd()
```

Comment:
`uv.cwd`

Reply:
if this pr works for `single_file_mode` the `root_dir` in there is nil then do some check